go.mod

```diff
@@ -101,10 +99,179 @@ require (
 	k8s.io/kube-openapi v0.0.0-20210305001622-591a79e4bda7
 	k8s.io/utils v0.0.0-20210819203725-bdf08cb9a70a
 	kubevirt.io/api v0.0.0-20220111180619-bd15f69822b9
-	kubevirt.io/client-go v0.49.0 // indirect
 	kubevirt.io/containerized-data-importer-api v1.41.0
 	kubevirt.io/kubevirt v0.49.0
 	sigs.k8s.io/cluster-api v0.4.4
 	sigs.k8s.io/kind v0.11.1
 	sigs.k8s.io/yaml v1.2.0
 )
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
```

Comment:
This change is the result of module graph pruning from 1.17
https://go.dev/doc/go1.17#graph-pruning